src/test/java/hudson/plugins/git/GitChangeSetBasicTest.java

```diff
@@ -130,4 +134,77 @@ public void testSwedishDate() {
     public void testSwedishTimestamp() {
         assertEquals(1363875404000L, genChangeSetForSwedCase(true).getTimestamp());
     }
+
+    @Test
+    public void testChangeLogTruncationWithShortMessage(){
+        GitChangeSet changeSet = GitChangeSetUtil.genChangeSet(true, false, true,
+                "Lorem ipsum dolor sit amet.",
+                true);
+        String msg = changeSet.getMsg();
+        Assert.assertTrue("title is correct", msg.contains("Lorem ipsum dolor sit amet."));
```

Comment:
Hamcrest assertions are easier to read and provide better error messages when they fail.  Could you switch this to use Hamcrest?
```suggestion
        assertThat(msg, containsString("Lorem ipsum dolor sit amet."));
```